util/envDynawo.sh

```diff
@@ -1715,7 +1717,7 @@ create_distrib_with_omc() {
     error_exit "$DYNAWO_DEPLOY_DIR does not exist."
   fi
   cd $DYNAWO_DEPLOY_DIR
-  zip -r -y $ZIP_FILE bin/ lib/ sources/
+  zip -r -y $ZIP_FILE bin/ lib/ sources/ data/
```

Comment:
Not sure I get why we are zipping a symbolic link that may not exist later?

Reply:
In fact it will store the files and not the symoblic links because of -y. I added it because I thought it is clearer to expose directly cases that are working with the distrib to test it.

Reply:
OK, however I think that "data" is not clear enough, I propose to rename it "tests" or "testcases"

Reply:
It is ok with me